nussl/audio_signal.py

```diff
@@ -108,16 +109,34 @@ def __str__(self):
     ##################################################
     #                   Plotting
     ##################################################
-
-    # def plot_time_domain(self):
-    #     """
-    #     Not implemented yet -- will raise and exception
-    #     Returns:
     #
-    #     """
-    #     signal = self.audio_data
-    #     for channel in num_channels
 
+    def plot_time_domain(self, x_label='time', save=False, name=None, output_path=None):
+        """
+        Parameters:
+            x_label (str): Label the x axis with time or samples
```

Comment:
If `x_label` can only be one of two values (time or samples) then it should be a bool. And in that case, maybe rename it to something more appropriate. Maybe `x_label_samples=False`, not sure though. Think of a better name...